articles/governance/policy/how-to/remediate-resources.md

```diff
@@ -89,15 +131,19 @@ $policyDef = Get-AzPolicyDefinition -Id '/providers/Microsoft.Authorization/poli
 # Get the reference to the resource group
 $resourceGroup = Get-AzResourceGroup -Name 'MyResourceGroup'
 
-# Create the assignment using the -Location and -AssignIdentity properties
-$assignment = New-AzPolicyAssignment -Name 'sqlDbTDE' -DisplayName 'Deploy SQL DB transparent data encryption' -Scope $resourceGroup.ResourceId -PolicyDefinition $policyDef -Location 'westus' -AssignIdentity
+# Get the existing user assigned managed identity ID
```

Comment:
This command creates the user assigned identity. We could either modify the comment to show that an identity is being created, or we can modify the powershell command to show a user assigned identity being fetched

Reply:
Updated